File: CCW알고리즘(외적)/백준_11758_CCW/CCW.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int x1, y1, x2, y2, x3, y3;
        x1 = s.nextInt();
        y1 = s.nextInt();
        x2 = s.nextInt();
        y2 = s.nextInt();
        x3 = s.nextInt();
        y3 = s.nextInt();

        int external = (((x2 - x1) * (y3 - y2)) - ((x3 - x2) * (y2 - y1)));
        if (external > 0)
            System.out.println("1");
        else if (external < 0)
            System.out.println("-1");
        else
            System.out.println("0");
    }
}

File: CCW알고리즘(외적)/백준_11758_CCW/CCW.kt
import java.util.*

fun main(args: Array<String>) {
    val s = Scanner(System.`in`)
    val x1 = s.nextInt()
    val y1 = s.nextInt()
    val x2 = s.nextInt()
    val y2 = s.nextInt()
    val x3 = s.nextInt()
    val y3 = s.nextInt()

    val external =(((x2 - x1) * (y3 - y2)) - ((x3 - x2) * (y2 - y1)))
    if (external > 0) print("1");
    else if (external < 0) print("-1");
    else print("0");

}


